pfcpiface/pfcpiface.go

```diff
@@ -91,7 +96,8 @@ func pfcpifaceMainLoop(upf *upf, accessIP string, sourceIP string) {
 		// if sourceIP is not set, fetch it from the msg header
 		if sourceIP == "0.0.0.0" {
```

Comment:
@krsna1729 @ajamshed I am not clear what we are doing here ? Can you please help. Since this is happening per packet and using the API getOutboundIP, want to know more.

Reply:
In case `cpiface` block does not mention neither `nb_src_ip` nor `nb_dst_ip` in `conf/upf.json`, then sourceIP of N4 is not known to begin with. It will use the PFCP Establishment request coming from the control plane to determine the IP address of sourceIP.

Reply:
this should be restructured to happen only once. probably by populating a N4Conn struct.